setup.yml

```diff
@@ -93,3 +93,15 @@
         - setup-all
 
     - role: galaxy/com.devture.ansible.role.playbook_runtime_messages
+
+- name: "Set up additional Jitsi JVB servers"
+  hosts: "jitsi_jvb_servers"
+  become: true
+
+  roles:
+    - role: galaxy/com.devture.ansible.role.playbook_help
+    - role: galaxy/com.devture.ansible.role.systemd_docker_base
+
+    - custom/matrix-base
+    - custom/matrix-jitsi
+    - custom/matrix-common-after
```

Comment:
Why not make this a completely different playbook file, other than `setup.yml`?

I suppose we could have:

- `playbook/matrix.yml` be what `setup.yml` is now
- `setup.yml` be a symlink to `playbook/matrix.yml`
- a new `playbook/jitsi_jvb.yml` setup file for `jitsi_jvb_servers`

Reply:
Sounds good to me

Reply:
I tried moving the contents of setup.yml into playbook/matrix.yml by adding an import playbook statement. e.g. in setup.yml

```
- name: Include the matrix playbook
  ansible.builtin.import_playbook: playbooks/matrix.yml
```

However this requires a lot of changes throughout the playbook to support change in directory structure as its using relative paths in a lot of places. For example, everywhere it refers to `custom/<role>` I've had to change to `roles/custom/<role>` Also I needed to move the `groups_vars` directory under playbooks so that it's alongside the playbook.

I've pushed another branch to my fork if you want to see what I mean "https://github.com/warrenbailey/matrix-docker-ansible-deploy/tree/multi-jvb-playbooks/playbooks"

@spantaleev  I just wanted to check my understand of what you're suggesting is right before I push the change to this PR?

Reply:
I did what I posted in my previous comment above and it worked:

- `mkdir playbook`
- `mv setup.yml playbook/matrix.yml`
- `ln -s playbook/matirx.yml setup.yml`
- `ansible-playbook -i inventory/hosts setup.yml --tags=setup-all`

----

Maybe you're `cd`-ing into the `playbook` directory and trying to use relative paths to your inventory, etc.? You probably shouldn't do that -- trying staying at the project root.

We could use `import_playbook`, but.. I don't see why we should do that when a symlink also works.

Reply:
Ah ok, I did miss understand then. I assumed you'd have to do an import playbook and wasn't sure symlinks would work like that with ansible.  

I'll get it sorted asap and pushed up to this PR. 👍